packages/theme/src/fonts/index.ts

```diff
@@ -0,0 +1,19 @@
+export const fonts = {
+  primary: {
```

Comment:
Can you also export `primary` and `secondary`? This can help with optimizing build during tree shaking

Reply:
Added individual exports